images/dotcms/ROOT/srv/templates/dotcms/CONF/dotcms-config-cluster.properties

```diff
@@ -22,7 +22,8 @@ REMOVE_INACTIVE_CLUSTER_SERVER_CRON_EXPRESSION=0/30 * * ? * * *
 ES_HOSTNAME={{ .Env.PROVIDER_ELASTICSEARCH_DNSNAMES }}
 ES_AUTH_BASIC_USER={{ with .Env.PROVIDER_ELASTICSEARCH_AUTH_BASIC_USER }}{{ . }}{{ else }}admin{{ end }}
 ES_AUTH_BASIC_PASSWORD={{ with .Env.PROVIDER_ELASTICSEARCH_AUTH_BASIC_PASSWORD }}{{ . }}{{ else }}admin{{ end }}
-
+ES_PORT={{ with .Env.PROVIDER_ELASTICSEARCH_PORT_HTTP }}{{ . }}{{ else }}9200{{ end }}
```

Comment:
Isn't this taken care of by https://github.com/dotCMS/docker/blob/master/images/dotcms/ROOT/srv/utils/config-defaults.sh#L95 ?